src/layers/relu.rs

```diff
@@ -0,0 +1,45 @@
+use std::{collections::HashMap, rc::Rc, vec};
```

Comment:
Why do you need this as a separate layer? Is it not fused?

Reply:
![image](https://github.com/ddkang/zkml/assets/53390631/d1762a9a-8093-435d-bbbd-c694f812464f)
It dis not fused.